### DISCUSSION TITLE
MVVM Helpers migrations

### DISCUSSION THREAD

**Discussion by IeuanWalker** [CommunityToolkit/Maui#417] [New Feature Discussions]
Just wondering if the grouping class from xamarin toolkit is going to be copied over here for maui?

https://github.com/xamarin/XamarinCommunityToolkit/blob/main/src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Grouping.shared.cs

**Comment by TheCodeTraveler** [comment]
Not into CommunityToolkit.Maui, no. 

That feature would go into the MVVM Toolkit, [CommunityToolkit.MVVM](https://github.com/CommunityToolkit/dotnet)

**Comment by CliffAgius** [comment]
Closed as answered
